packages/@okta/vuepress-site/guides/sign-into-web-app/require-authentication/aspnet/reqauthspecific.md

```diff
@@ -1,4 +1,4 @@
-Use the `Authorize` attribute on controllers or actions to require a logged-in user:
+Use the `[Authorize]` attribute on controllers or actions to require a logged-in user:
```

Comment:
just noticed this so thought I'd add a note:

Use the `[Authorize]` attribute on controllers or actions to require a signed-in user:

Reply:
Good catch 👍